src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php

```diff
@@ -119,4 +119,25 @@ protected function compileEndAuth()
     {
         return '<?php endif; ?>';
     }
+ 
+    /**
+     * Compile the if-guest statements into valid PHP.
+     *
+     * @param  string|null  $guard
+     * @return string
+     */
+    protected function compileGuest($guard = null)
+    {
+        return "<?php if(auth()->guard{$guard}->guest()): ?>";
```

Comment:
Seems taylor changed it...

https://github.com/laravel/framework/commit/a11b44aeef5acdca514c8da26cb9c8d2596fc7ca